docs/source/remote_data.rst

```diff
@@ -409,3 +410,17 @@ Company filing index from EDGAR (SEC).
     import pandas_datareader.data as web
     ed = web.DataReader('full', 'edgar-index')
     ed[:5]
+
+.. _remote_data.tsp:
+
+TSP Fund Data
+=============
+
+Download mutual fund index prices for the TSP.
+
+.. ipython:: python
+
+    import pandas_datareader.tsp as tsp
+    tspreader = tsp.TSPReader(start='2015-10-1',end='2015-12-31')
```

Comment:
a space before `end` and after comma is missing

    tspreader = tsp.TSPReader(start='2015-10-1', end='2015-12-31')

see https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements